src/openforms/formio/dynamic_config/referentielijsten.py

```diff
@@ -14,7 +15,7 @@
 
 def fetch_options_from_referentielijsten(
     component: Component, submission: Submission
-) -> list[tuple[str, str]] | None:
+) -> list[list[str, str]] | None:
```

Comment:
let's make the returned data actually tuples instead. A list implies arbitrary length, while a tuple implies a known/fixed length of each item. The type annotation is correct, the implementation not.

Can you check if we can add this file to the type checker config so that it's checked in CI in the future?